src/v/datalake/BUILD

```diff
@@ -99,3 +116,27 @@ redpanda_cc_library(
         "@seastar",
     ],
 )
+
+redpanda_cc_library(
+    name = "values_protobuf",
+    srcs = [
+        "values_protobuf.cc",
+    ],
+    hdrs = [
+        "values_protobuf.h",
+    ],
+    include_prefix = "datalake",
+    visibility = [":__subpackages__"],
```

Comment:
consider adding a default visibility on the package, then adding exceptions to each individual rule.